extensions/amp-selector/amp-selector.md

```diff
@@ -27,7 +27,7 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
-    <td>All</td>
```

Comment:
Right now per the code, this supports all layouts (does not really care about the layout.) We should just leave it as is.